package.json

```diff
@@ -21,8 +21,10 @@
     "test:e2e": "jest --config ./test/jest-e2e.json",
     "prepare": "husky install",
     "config": "npm run command config",
-    "register": "nest start --config nest-commander.json -- register",
-    "unregister": "nest start --config nest-commander.json -- unregister",
+    "api-extension-list": "nest start --config nest-commander.json -- api-extension-list",
+    "api-extension-add": "nest start --config nest-commander.json -- api-extension-add",
+    "api-extension-update": "nest start --config nest-commander.json -- api-extension-update",
+    "api-extension-delete": "nest start --config nest-commander.json -- api-extension-delete",
```

Comment:
Change `"version": "0.0.1"` to `"version": "2.0.0"` to be visible on GitHub badges.